lib/views/name.erb

```diff
@@ -0,0 +1,7 @@
+<%user = User.create(:name => 'Jennifer')
+    user.save %>
```

Comment:
Do not make database calls or adjustments in the views. This kind of logic belongs in the controller and stored in instance variables `@user`.